addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -1,114 +1,153 @@
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="clientSecret" type="text">
+            <label>Client Secret</label>
+            <description>Client Secret provided for the application you created</description>
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="username" type="text">
+            <label>Username</label>
+            <description>Your Netatmo API username (email)</description>
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="password" type="text">
+            <label>Password</label>
+            <description>Your Netatmo API password</description>
+            <required>true</required>
+            <advanced>false</advanced>
+            <context>password</context>
+        </parameter>            
+                       
+        <parameter name="readStation" type="boolean" required="true">
+            <label>Access Weather Station</label>
+            <description>Read weather station's data.</description>
+            <default>true</default>
+        </parameter>
+        
+        <parameter name="readHealthyHomeCoach" type="boolean" required="true">
+            <label>Access Healthy Home Coach</label>
+            <description>Read healthy home coach's data.</description>
+            <default>false</default>
+        </parameter>        
+            
+        <parameter name="readThermostat" type="boolean" required="true">
+            <label>Access Thermostat</label>
+            <description>Read and Write thermostat's data.</description>
+            <default>false</default>
+        </parameter>
+        
+        <parameter name="readWelcome" type="boolean" required="true">
+            <label>Access Welcome camera</label>
+            <description>Read and Access Welcome camera's data.</description>
+            <default>false</default>
+        </parameter>
+        
+        <parameter name="writeWelcome" type="boolean" required="true">
+            <label>Write Welcome camera</label>
+            <description>Write Welcome camera's data (this requires write_camera scope request on dev.netatmo.com).</description>
+            <default>false</default>
+        </parameter>
+        
+        <parameter name="webHookURL" type="text" required="false">
+            <label>Webhook URL</label>
+            <description>IP and port to access OH2 server from Internet.</description>
+        </parameter>                   
+                                                         
+    </config-description>
+
+    <config-description uri="thing-type:netatmo:device">
+        <parameter name="id" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})">
```

Comment:
I'm afraid, yes because of the introduction of Welcome that does not strictly have a parent / child dependance of modules. And as this binding grows as Netatmo API and product grows, I think it's better to be coherent.

Reply:
:+1:

Reply:
Indentation is wrong.